src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.ts

```diff
@@ -61,7 +61,7 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 
 	@HostListener('document:keydown', ['$event'])
 	onkeydown(e: KeyboardEvent) {
-		if (this.isActive()) {
+		if (this.isActive() && this.isWYSIWYGMode()) {
```

Comment:
Is this listener only for WYSIWYG? Why we don't check if it's in Markdown mode as well?

Reply:
@barbaravaldez looking at the code I believe it is; we shouldn't override any keyboard shortcuts that already exist for editors (i.e. they already have undo/redo defined)